packages/orbit-design-tokens/src/dictionary/formatters/jsFoundation.ts

```diff
@@ -27,7 +27,7 @@ import { falsyString } from "../utils/string.js";
 const genericFactory = (allProperties: DesignToken[], platform: "javascript" | "typescript") => {
   const upperFirst = determinateUpperFirst(platform);
   const withExport = platform !== "javascript";
-  const variableType = platform !== "javascript" ? "type" : "const";
+  const variableType = platform !== "javascript" ? "interface" : "const";
```

Comment:
Could you explain the reason for this change? Why are interfaces preferable to types?

Reply:
Because we have a eslint rule *"@typescript-eslint/consistent-type-definitions": ["error", "interface"]* and when it was creating types it had to be formatted with eslint and prettier to have it correct.

Reply:
Okay, but why is this change happening now? What does it have to do with the goal of this PR?

Reply:
I personally do not mind having such kind of improvements alongside, if there aren't too many and they committed well. Will do it separately then

Reply:
It's not about you, @mainframev.

The PR has +2,500 -1,900 lines of code, that's a lot of changes to review.

Anything that is off-topic should be avoided, it just adds noise and confusion to what you're trying to accomplish.

It makes reviewing harder, it increases the risk of this PR being delayed, etc.